<ui version="4.0" >
 <class>createTorrentDialog</class>
 <widget class="QDialog" name="createTorrentDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Torrent Creation Tool</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="createTorrent_title" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>27</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>Torrent file creation</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_input" >
     <property name="text" >
      <string>File or folder to add to the torrent:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="textInputPath" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="addFile_button" >
       <property name="text" >
        <string>Add a file</string>
       </property>
       <property name="icon" >
        <iconset resource="icons.qrc" >:/Icons/add_file.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addFolder_button" >
       <property name="text" >
        <string>Add a folder</string>
       </property>
       <property name="icon" >
        <iconset resource="icons.qrc" >:/Icons/add_folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="lbl_announce_url" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>102</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="text" >
          <string>Announce urls (trackers):</string>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="urlSeeds_lbl" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>101</height>
          </size>
         </property>
         <property name="text" >
          <string>Web seeds urls (optional):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_comment" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>102</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>102</height>
          </size>
         </property>
         <property name="text" >
          <string>Comment (optional):</string>
         </property>
         <property name="buddy" >
          <cstring>txt_comment</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="trackers_list" >
           <property name="selectionMode" >
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addTracker_button" >
             <property name="minimumSize" >
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="icon" >
              <iconset resource="icons.qrc" >:/Icons/skin/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeTracker_button" >
             <property name="minimumSize" >
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="icon" >
              <iconset resource="icons.qrc" >:/Icons/skin/remove.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="URLSeeds_list" >
           <property name="selectionMode" >
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addURLSeed_button" >
             <property name="minimumSize" >
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="icon" >
              <iconset resource="icons.qrc" >:/Icons/skin/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeURLSeed_button" >
             <property name="minimumSize" >
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="icon" >
              <iconset resource="icons.qrc" >:/Icons/skin/remove.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="txt_comment" >
         <property name="maximumSize" >
          <size>
           <width>421</width>
           <height>102</height>
          </size>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="txtPieceSize" >
       <property name="text" >
        <string>Piece size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboPieceSize" >
       <property name="currentIndex" >
        <number>3</number>
       </property>
       <item>
        <property name="text" >
         <string>32 KiB</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>64 KiB</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>128 KiB</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>256 KiB</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>512 KiB</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>1 MiB</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>2 MiB</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>4 MiB</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="check_private" >
     <property name="text" >
      <string>Private (won't be distributed on DHT network if enabled)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkStartSeeding" >
     <property name="text" >
      <string>Start seeding after creation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="progressLbl" >
     <property name="text" >
      <string>Progress:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton" >
       <property name="text" >
        <string>Create and save...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>createTorrentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>355</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
